sdk/eventhub/azure-eventhub/tests/pyamqp_tests/unittest/test_message.py

```diff
@@ -63,3 +63,18 @@ def test_message_properties():
     value = AmqpMessageProperties()
     value.user_id = bytearray(b"\nweird\0user\1id\0\t")
     assert value.user_id == b"\nweird\0user\1id\0\t"
+
+
+def test_amqp_sequence_value():
+    message = Message(sequence=[{"key": "value"}, "test_message", 123, (1,2)])
+
+    output = bytearray()
+    encode_payload(output, message)
+
+    message = decode_payload(memoryview(output))
+    output = bytearray()
+
+
+    sequence = message.sequence
+    assert type(sequence) == list
+    assert sequence[0][0] == {b"key": b"value"}
```

Comment:
output of pyamqp Message sequence changes

in current SDK this would be sequence[0] now list of lists


We cannot have the for loop in _encode or will we get `azure.eventhub.exceptions.EventDataSendError: The format code '160' at frame buffer offset '283' is invalid or unexpected.`